pkgs/development/tools/phpactor/default.nix

```diff
@@ -63,7 +67,8 @@ stdenvNoCC.mkDerivation {
     mkdir -p $out/share/php/phpactor $out/bin
     cp -r . $out/share/php/phpactor
     cp -r ${vendor}/vendor $out/share/php/phpactor
-    ln -s $out/share/php/phpactor/bin/phpactor $out/bin/phpactor
+    makeBinaryWrapper ${php}/bin/php "$out/bin/phpactor" \
```

Comment:
```suggestion
    makeWrapper ${php}/bin/php "$out/bin/phpactor" \
```

Reply:
I'm still a beginner at this, so please excuse my ignorance.

I see that `makeWrapper` is an alias for `makeBinaryWrapper`, but isn't it a bit confusing to use the alias which is named the same as a different function?

From 62245943aa12a it seems that the shell function was renamed to `makeBinaryWrapper`, and `makeWrapper` was kept as an alias for backwards compatibility

Reply:
No problem :)

The implementation of that tool can be changed, therefore I would recommend using the suggestion I made.